include/container.h

```diff
@@ -81,6 +82,11 @@ struct sway_container {
 	char *class;
 	char *app_id;
 
+	// Used by output containers to keep track of swaybar/swaybg child
+	// processes.
+	list_t *bar_pids;
+	pid_t bg_pid;
+
```

Comment:
This struct is getting kind of out of control. Half of the properties only apply to a certain kind of swayc (i.e. C_OUTPUT, in this case). Can we figure out a good way of splitting these up?